finality-aleph/src/new_network/bare.rs

```diff
@@ -0,0 +1,42 @@
+use crate::new_network::{DataNetwork, SendError};
```

Comment:
Please, think of a better name for this file. I don't know what to expect in the file named like this

Reply:
Changed to `component` (as in `ComponentNetwork`, since this is just components for a proper `DataNetwork`), not sure whether this is better.